.eslintrc.json

```diff
@@ -13,8 +13,5 @@
 	},
 	"rules": {
 		"@typescript-eslint/no-unused-vars": ["error", {"argsIgnorePattern": "^_"}]
-	},
-	"plugins": [
-		"@typescript-eslint"
-	]
```

Comment:
Any reason why this was removed? Is it not needed?

We might want to add in things that we want to ignore so that if eslint is ran in the same directory of this file, it only picks up src and test or whatever. Or we can just include what we want instead of ignoring what we don't want.

Reply:
That might have been a mistake. I can add it back in. I'll also add ignore entries to either this file or the `.eslintignore` file.